src/main/java/com/skynest/uitesting/tests/BaseTest.java

```diff
@@ -0,0 +1,100 @@
+package com.skynest.uitesting.tests;
+
+import com.github.javafaker.Faker;
+import com.skynest.uitesting.pages.LoginPage;
+import com.skynest.uitesting.pages.RegistrationPage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+import java.time.Duration;
+
+import static com.skynest.uitesting.utils.BaseHelper.randomEmail;
+
+public class BaseTest {
+    public static final int TIMEOUT = 20;
+    public WebDriver driver;
+    public WebDriverWait wait;
+    public String browser = "chrome";
+    public Faker faker = new Faker();
+    public String firstName;
```

Comment:
As mentioned, create a User class with all the necessary fields and perhaps create a static factory method that would return a randomly generated but valid User instance and put all the faker logic there,